machine-learning/lambda-v2/README.md

```diff
@@ -54,7 +62,7 @@ The following configuration is applicable for both the `com.lambda.DLRImageClass
 
 - **PublishResultsOnTopic** (Optional) : The topic on which you want to publish the inference results. If you modify this value, then you must also modify the value of resources in the accessControl parameter to match your custom topic name.
 
-    Default: `gg/sageMakerEdgeManager/<inference-type>`
+    Default: `""`
 
 - **ImageDirectory** (Optional) : The path of the folder on the device where inference components read images. You can modify this value to any location on your device to which you have read/write access.
```

Comment:
Minor style changes:

> - **ImageDirectory** - (Optional) The path of the folder on the device where this component reads images. The system user that runs these ML inference components must have read/write access to the folder that you specify.

Also, should this parameter be required? Otherwise, what's its default value?

Reply:
Optional. Default one is provided through the source code - https://github.com/aws-greengrass/aws-greengrass-component-examples/blob/main/machine-learning/lambda-v2/lambda-artifacts/com.lambda.Container.DLRImageClassification/1.0.0/image_classification/config_utils.py#L34

Reply:
Can we add the default to the readme? `<greengrass-root>/artifacts/sample_images`